vulcan/lib/client/jsx/components/workflow/browser.jsx

```diff
@@ -30,24 +32,52 @@ const errorDiv = (
 
 export default function Browser() {
   const invoke = useActionInvoker();
-  const browserState = useReduxState(browserStateOf());
+  const [workflows, setWorkflows] = useState(null);
```

Comment:
Should probably use the context hook here instead...will refactor as we get into UI components.